pywbem/config.py

```diff
@@ -93,3 +93,17 @@
 #:
 #: *New in pywbem 0.12.*
 SEND_VALUE_NULL = True
+
+#: Backward compatibility option that controls creating CIMInstanceNames with
```

Comment:
Instead of using plural on a class name, use a link on the class name:
```
... creating :class:`~pywbem.CIMInstanceName` objects with ...

Reply:
DONE